docker-compose.yml

```diff
@@ -1,5 +1,3 @@
-version: "3.6"
-
```

Comment:
Has the recommendation to specify version been removed?

Reply:
Yeah for the latest versions of the CLI there is a message saying it could be removed (it hasn't really been needed for >3 AFAIK)